src/app/clusters/level-control/level-control.cpp

```diff
@@ -139,6 +146,33 @@ static void timerCallback(System::Layer *, void * callbackContext)
     emberAfLevelControlClusterServerTickCallback(static_cast<EndpointId>(reinterpret_cast<uintptr_t>(callbackContext)));
 }
 
+static uint32_t calculateNextWaitTimeMs(NextWaitTimeState * nextwaitTimeState, uint32_t delayMs)
+{
+    chip::System::Clock::Timestamp latency;
```

Comment:
Why not declare this where it's actually used?

Reply:
Done